portal/src/locale-data/en.json

```diff
@@ -691,9 +692,9 @@
   "AuthenticatorType.primary.password": "Password",
   "AuthenticatorType.primary.oob-otp-email": "Passwordless via Email",
   "AuthenticatorType.primary.oob-otp-phone": "Passwordless via Phone",
-  "AuthenticatorType.secondary.totp": "Authenticator Apps / Devices",
-  "AuthenticatorType.secondary.oob-otp-email": "Verify via Email",
-  "AuthenticatorType.secondary.oob-otp-phone": "Verify via Phone",
+  "AuthenticatorType.secondary.totp": "OTP Code via Google Authenticator/Authy (TOTP)",
```

Comment:
This copywriting is truncated due to its increased length. Please adjust the table to allocate more space to 2nd column